.travis.yml

```diff
@@ -11,8 +10,6 @@ os:
   - linux
   - osx
 
-osx_image: xcode61
```

Comment:
ok.
By the way it is very fine the potential of DGtal in xcode (in the first use Xcode takes a little time to parse headers but after it is nice)...